ts/screens/messages/MessagesArchiveScreen.tsx

```diff
@@ -2,12 +2,10 @@ import * as React from "react";
 import { useMessageOpening } from "../../components/messages/hooks/useMessageOpening";
 import MessagesArchive from "../../components/messages/MessagesArchive";
 import { upsertMessageStatusAttributes } from "../../store/actions/messages";
-import { useIODispatch, useIOSelector } from "../../store/hooks";
-import { allArchiveMessagesSelector } from "../../store/reducers/entities/messages/allPaginated";
+import { useIODispatch } from "../../store/hooks";
 import { UIMessage } from "../../store/reducers/entities/messages/types";
 
 const MessagesArchiveScreen = () => {
```

Comment:
This screen is 96% equal to MessagesInboxScreen (also consider the inner component that can be unified - see related comment). Shall we try to merge them?